### ISSUE TITLE
Android Icon Missing

### ISSUE THREAD

**Issue by francwalter** [ash12345678987654321/android_mouse#1]
The Android App has no icon.
The default Android icon is used.

Frank

**Comment by ash12345678987654321** [comment]
cant replicate, cant fix
make sure ur using cutting edge

other than that i really dk what to do

**Comment by francwalter** [comment]
Yes, I installed the last version. Maybe it is intended. This is the icon I have in my app-drawer:
![grafik](https://user-images.githubusercontent.com/927269/81723741-98003b00-9483-11ea-9fcd-5af36b69243c.png)
Is it the icon which should be the app's icon?

**Comment by francwalter** [comment]
... and is there a keyboard layout with all keys which is already complete and I could download and install?
Or some xml-file to import? Any way to import a layout?

**Comment by ash12345678987654321** [comment]
No thats not the icon its supposed to be.

Anyways theres no way to import layouts now. You have to create one yourself. 

Btw im probably not going to debug or add new features this app anymore as i made it for some school project.

**Comment by ash12345678987654321** [comment]
On a side note if you dont mind using bluetooth you can probably use this.


https://github.com/raghavk92/Kontroller

Its probably alot less buggy than my code and theres probably less latency since my code uses (unencrypted) TCP protocols which also makes it really insecure.

**Comment by francwalter** [comment]
Bluetooth is not possible for me to use, the computer is too far away for it, tested it already.
Not stable at all.
But thanks anyway. 

> Btw im probably not going to debug or add new features this app anymore as i made it for some school project.

OK, understand now. Thanks

**Comment by umiyuikaiteitan** [comment]
> im probably not going to debug or add new features this app anymore
